### PR TITLE
Chore update to use composability API

### PR DESCRIPTION
**Pull request by Xantier**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -14,7 +14,15 @@
  * limitations under the License.
  */
 
-export { githubInsightsPlugin as plugin } from './plugin';
+export {
+  githubInsightsPlugin as plugin,
```
Comment (iain-b) [inline src/index.ts:18]: Should we export the named plugin too ? i.e. githubInsightsPlugin 
That's the new pattern right ?

Comment (Xantier) [reply src/index.ts:18]: Yup, good call. I'll modify this PR a bit to be more involved so we can conform to the new entity-less approach as well.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@roadiehq/backstage-plugin-github-insights",
-  "version": "0.3.3",
+  "version": "0.4.0",
```
Comment (Xantier) [inline package.json:3]:
```suggestion
  "version": "1.0.0",
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@roadiehq/backstage-plugin-github-insights",
-  "version": "0.3.3",
+  "version": "1.0.0",
```
Comment (iain-b) [inline package.json:3]: This is backwards compatible, no?

**Review by iain-b — APPROVED** [review]

**Comment by iain-b** [comment]
Fixes #125

**Pull request merged**